python/sbp/client/drivers/pyserial_driver.py

```diff
@@ -20,21 +23,29 @@ class PySerialDriver(BaseDriver):
   Parameters
   ----------
   port : string
-    Path to port to read SBP messages from.
+    URI to port to read SBP messages from. Accepts the following types
+    of URL's. Accepts the following types of URLs:
+    - rfc2217://<host>:<port>[/<option>[/<option>]]
+    - socket://<host>:<port>[/<option>[/<option>]]
+    - loop://[<option>[/<option>]]
+    and device names, such as /dev/ttyUSB0 (Linux) and COM3 (Windows). See
+    http://pyserial.sourceforge.net/pyserial_api.html#urls for more details.
   baud : int
```

Comment:
Should 'baud' be optional, now that you can connect to things for which it doesn't make sense?